### DISCUSSION TITLE
Single Select with numeric values in graph's Y-axis

### DISCUSSION THREAD

**Discussion by Dyljyn** [community/community#15113] [Projects and Issues]
I have a single select for "Story Points" that have numeric values (e.g. 1, 2, 3, 5, etc.) and I'd like to display a graph with the sum of these story points. However, the options like "Sum of a field", "Average of a field", etc. for the Y-axis currently seem to only support fields with the type "number".

**Comment by abaldwinls** [comment]
Looking for the same thing!

**Comment by nanoni17728** [comment]
Same here!
